Flow.Launcher.Core/Resource/Theme.cs

```diff
@@ -244,7 +244,7 @@ public void AddDropShadowEffectToCurrentTheme()
             effectSetter.Property = Border.EffectProperty;
             effectSetter.Value = new DropShadowEffect
             {
-                Opacity = 0.9,
```

Comment:
Could you provide a screenshot for people to check the effect of this change?

Reply:
![image](https://user-images.githubusercontent.com/6903107/134942072-0d8bfe89-40d4-4756-b357-8cccddee1a5f.png)
It turned into a softer shadow. I tried to give it a feeling that is commonly used in modern Windows.